moodlecli/utils.py

```diff
@@ -99,6 +100,19 @@ def create_or_get_user(moodle_client, firstname, lastname, email, auth):
     return user_id
 
 
+def stylize_courses(courses_json):
+    t = PrettyTable()
```

Comment:
Super minor: This is purely subjective, but what do you think about making the `fullname` column left aligned instead of centered?

Reply:
The quick alignment function I saw aligned every column, and when I tried it it was ugly. I'm sure there's a way to align only the first col - I agree that might be nice.

Reply:
Would something like `t.align["fullname"] = "l"` work?